sos/report/plugins/filesys.py

```diff
@@ -35,6 +35,21 @@ class Filesys(Plugin, DebianPlugin, UbuntuPlugin, CosPlugin):
     ]
 
     def setup(self):
+        self.add_forbidden_path([
+            # cifs plugin
+            '/proc/fs/cifs',
```

Comment:
We collect just some stuff from that dir, but I guess other stuff is not important (even for cifs plugin itself, so why important for filesys plugin?)